satellite/rewards/rewards.go

```diff
@@ -40,6 +40,19 @@ type NewOffer struct {
 	Type   OfferType
 }
 
+// FormatPartnerName formats partner's name into combination of its partnerID and name
+func (o NewOffer) FormatPartnerName() string {
+	if o.Type != Partner {
+		return o.Name
+	}
+
+	partnerInfo := PartnerInfo{
```

Comment:
Maybe there's better way to do this? @coyle

Reply:
That might be adding too much complexity. But `LoadPartnerInfos()[o.Name]` should already be of type `PartnerInfo` meaning there probably isn't a need for the new creation of a `PartnerInfo` here